ENIGMAsystem/SHELL/Universal_System/Extensions/Paths/path_functions.cpp

```diff
@@ -60,6 +60,12 @@ namespace enigma_user
 
 void path_start(unsigned pathid, cs_scalar speed, unsigned endaction, bool absolute)
 {
+    #ifndef PATH_EXT_SET
+        return;
+    #endif
+
+    return;  //function can cause crashes atm, until extension variables fixed
```

Comment:
Are these actually broken still? There is no early return in our current version and I haven seen a report of crashing.